nucypher/blockchain/eth/agents.py

```diff
@@ -806,6 +828,17 @@ def lock_restaking(self, release_period: int) -> dict:
         # TODO: Handle ReStakeLocked event (see #1193)
         return receipt
 
+    def set_winding_down(self, value: bool) -> dict:
```

Comment:
`PreallocationEscrowAgent` has some code that looks same as in `StakingEscrowAgent`, but it different because uses beneficiary address, also `PreallocationEscrowAgent` and `StakingEscrowAgent` have different contracts 
I guess it's possible that `PreallocationEscrowAgent` will reuse `StakingEscrowAgent`  but it's not in scope of this PR